lib/heimdall-utils.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+function isInstrumentedBuild() {
+  return process.argv.indexOf('--instrument');
```

Comment:
This is incorrect, should have done `> -1` at the end here, but I’ll refactor to `.includes`.